verl/experimental/agent_loop/single_agent_loop.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Bytedance Ltd. and/or its affiliates
```

Comment:
Can we rename this file to single_turn_agent_loop? `single_agent_loop` is confusing.